### PR TITLE
docs(observability): how to activate cockpit MTA-2314

### PR DESCRIPTION
**Pull request by nedaehm**
### Your checklist for this pull request

- [ ] Check that the commit messages match our requested structure.
- [ ] Name your pull request according to the [contribution guidelines](../docs/CONTRIBUTING.md).

### Description
Please describe your pull request.

### CODE REVIEW

File: observability/cockpit/how-to/activate-cockpit.mdx [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+meta: How to activate your Cockpit
+  title: How to activate your Cockpit
+  description: This page shows you how to activate your Cockpit
+content:
+  h1: How to activate your Cockpit
+  paragraph: This page shows you how to activate your Cockpit
+categories:
+  - observability
+dates:
+  validation: 2022-10-17
```
Comment (bene2k1) [inline observability/cockpit/how-to/activate-cockpit.mdx:11]:
```suggestion
  validation: 2022-10-30
```

make these dates unique :)

File: observability/cockpit/how-to/activate-cockpit.mdx [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+
+<Message type="requirement">
+
+ You have an account and are logged into the [Scaleway console](https://console.scaleway.com)
+
+</Message>
+
+# How to activate your Cockpit
+
+1. Log into your [Scaleway Console](https://console.scaleway.com/) account.
```
Comment (bene2k1) [inline observability/cockpit/how-to/activate-cockpit.mdx:25]:
```suggestion
1. Log into your [Scaleway console](https://console.scaleway.com/) account.
```

File: observability/cockpit/how-to/activate-cockpit.mdx [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+  h1: How to activate your Cockpit
+  paragraph: This page shows you how to activate your Cockpit
+categories:
+  - observability
+dates:
+  validation: 2022-10-17
+  posted: 2022-10-30
+---
+
+This page shows you how to activate your Scaleway Cockpit using the [Scaleway console](https://console.scaleway.com/), to start monitoring your Scaleway products.
```
Comment (bene2k1) [inline observability/cockpit/how-to/activate-cockpit.mdx:15]:
```suggestion
This page shows you how to activate and monitor your Scaleway products from your Cockpit using the [Scaleway console](https://console.scaleway.com/).
```
maybe like this to reduce the number of SCW in the sentence?

File: observability/cockpit/how-to/activate-cockpit.mdx [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+
+<Message type="requirement">
+
+ You have an account and are logged into the [Scaleway console](https://console.scaleway.com)
+
+</Message>
+
+# How to activate your Cockpit
+
+1. Log into your [Scaleway console](https://console.scaleway.com/) account.
```
Comment (RoRoJ) [inline observability/cockpit/how-to/activate-cockpit.mdx:25]: Unnecessary step I think

Comment (nedaehm) [inline observability/cockpit/how-to/activate-cockpit.mdx:25]:
```suggestion
```

File: observability/cockpit/how-to/activate-cockpit.mdx [hunk 5]
```diff
@@ -0,0 +1,34 @@
...
+
+2. Click **Cockpit** in the side menu.
```
Comment (nedaehm) [inline observability/cockpit/how-to/activate-cockpit.mdx:27]:
```suggestion
1. Click Cockpit in the Observability section of the [console](https://console.scaleway.com/) side menu. The **Cockpit** overview page displays.
```

File: observability/cockpit/how-to/activate-cockpit.mdx [hunk 6]
```diff
@@ -0,0 +1,34 @@
...
+
+3. Click **Activate** to activate your Cockpit. Your Cockpit will be ready in a matter of minutes.
```
Comment (nedaehm) [inline observability/cockpit/how-to/activate-cockpit.mdx:29]:
```suggestion
2. Click **Activate** to activate your Cockpit. Your Cockpit will be ready in a matter of minutes.
```

File: observability/cockpit/how-to/activate-cockpit.mdx [hunk 7]
```diff
@@ -0,0 +1,33 @@
...
+
+This page shows you how to activate and monitor your Scaleway products from your Cockpit using the [Scaleway console](https://console.scaleway.com/).
+
+<Message type="requirement">
+
+ You have an account and are logged into the [Scaleway console](https://console.scaleway.com)
+
+</Message>
+
+# How to activate your Cockpit
```
Comment (nedaehm) [inline observability/cockpit/how-to/activate-cockpit.mdx:23]:
```suggestion
```

**Review by RoRoJ — APPROVED** [review]

File: observability/cockpit/how-to/activate-cockpit.mdx [hunk 8]
```diff
@@ -0,0 +1,33 @@
...
+
+
+1. Click Cockpit in the Observability section of the [console](https://console.scaleway.com/) side menu. The **Cockpit** overview page displays.
```
Comment (RoRoJ) [inline observability/cockpit/how-to/activate-cockpit.mdx:26]:
```suggestion
1. Click **Cockpit** in the Observability section of the [console](https://console.scaleway.com/) side menu. The **Cockpit** overview page displays.
```

**Review by ldecarvalho-doc — APPROVED** [review]

**Pull request merged**